helm/hpcc/templates/dali.yaml

```diff
@@ -64,18 +81,8 @@ spec:
     spec:
       {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" $dali.name "type" "dali") | indent 6 }}
       serviceAccountName: "hpcc-default"
-      {{- $sashaPlanes := list }}
-      {{- if not ( $sashaServices.disabled ) -}}
-      {{- range $sashaName, $_sasha := $dali.services -}}
-      {{- $sasha := ($_sasha | default dict) -}}
-      {{- if and (not $sasha.disabled) (hasKey $sasha "plane") }}
-        {{- $sashaStoragePlane := $sasha.plane | default (include "hpcc.getFirstPlaneForCategory" (dict "root" $ "category" "sasha")) }}
-        {{- $_ := append $sashaPlanes $sashaStoragePlane }}
-      {{- end }}
-      {{- end }}
-      {{- end }}
       initContainers: 
-        {{- include "hpcc.changePlaneMountPerms" (dict "root" $ "includeCategories" (list "dali") "includeNames" (uniq $sashaPlanes)) | indent 8 }}
+      {{- include "hpcc.changePlaneMountPerms" (dict "root" $ "includeCategories" $tmpDaliScope.aggregatePlaneCategories "includeNames" $tmpDaliScope.aggregateSashaNamedPlanes) | indent 6 }}
```

Comment:
Slightly suprised this works with the leading {{- and indent.  Would nindent be better?  Is the change from 8 to 6 correct?

Reply:
I am slightly surprised to now you point it out, {{- same as before, but.. yes I think needs to be | indent.

8 -> 6 was intentinal, either 'works' but indenting the list at same level is more conventional (and same as 'containers' indendentation below it)

Not new or related to this PR, but I'll change to nindent.

Reply:
ah I see why it worked, by luck.. it was because the helper func. changeMountPerms begins with a block of comments.
So this (and other places) were wrong and swallowed up the newline, meaning the 1st comment of changeMountPerms anded up on the initContainers: line, which luckily didn't cause parsing problems.